### DISCUSSION TITLE
Merge models Page with Node

### DISCUSSION THREAD

**Discussion by jrief** [django-cms/django-cms#7873] [Polls]
Up to version 3, there where 2 `Page` objects per `Node` object, hence a separation of these models made sense. In version 4 this is not the case anymore. For each `Node` there is one and only one `Page` object. For simplicity and performance it therefore would be a good idea to merge these two models into one; this eliminates one INNER JOIN on many queries.

**Comment by fsbraun** [comment]
I support this for django CMS 4.2. Tricky bit will be backward code compatibility.

**Comment by fsbraun** [comment]
Implemented with #7947
